Add dropdown_multiple interface & rename multiselect_list to select_list_multiple

rijkvanzanten opened the pull request.

rijkvanzanten committed:
Add basic dropdown interface

rijkvanzanten committed:
Add upgrade to select2 as option

rijkvanzanten committed:
Add dropdown_multiple interface

rijkvanzanten committed:
Override select2 width

rijkvanzanten committed:
Rename multiselect_list to select_list_multiple

benhaynes reviewed (commented):
let's rename the interfaces and then we're good!

rijkvanzanten committed:
Merge branch 'version/6.4' into dropdown

rijkvanzanten committed:
Rename interfaces

dropdown_multiple > dropdown_multiselect
select_list_multiple > checkboxes
select_list > radio_buttons
